src/gt4py/cartesian/backend/pyext_builder.py

```diff
@@ -47,7 +47,8 @@ def get_cuda_compute_capability():
 
 def get_gt_pyext_build_opts(
     *,
-    debug_mode: bool = False,
+    opt_level: Literal["0", "1", "2", "3", "s"] = "3",
```

Comment:
Semantically I would do the following:
```suggestion
    opt_level: Literal["0", "1", "2", "3", "s"] = "3" if not debug_mode else "0",
```
I.e. keep the old behavior for `debug_mode`, but if you use additionally the opt_level, then overwrite the default opt_level.